projects/services/communities.py

```diff
@@ -31,7 +31,10 @@ def get_communities_feed(
 
     # If hidden from the main feed
     if unlisted is not None:
-        qs = qs.filter(unlisted=unlisted)
+        if unlisted:
+            qs = qs.filter(visibility=Project.Visibility.UNLISTED)
+        else:
+            qs = qs.exclude(visibility=Project.Visibility.UNLISTED)
```

Comment:
Worth double checking that we aren't missing how "not in main feed" should be slid into this.